server/src/main/java/org/elasticsearch/TransportVersions.java

```diff
@@ -281,6 +281,7 @@ static TransportVersion def(int id) {
     public static final TransportVersion IDP_CUSTOM_SAML_ATTRIBUTES = def(9_087_0_00);
     public static final TransportVersion JOIN_ON_ALIASES = def(9_088_0_00);
     public static final TransportVersion ILM_ADD_SKIP_SETTING = def(9_089_0_00);
+    public static final TransportVersion ESQL_PROFILE_INCLUDE_PLAN = def(9_090_0_00);
```

Comment:
I *had* said to auto-backport this. but it needs a transport version change. I should have know it does.

I *think* it's still worth it. Imagine someone giving us an 8.19 profile a year from now - we'll be happy that you added the plan....